### PR TITLE
RuCapitallLettersAfterBracketRule and MultiLinksRule added

### PR DESCRIPTION
**Pull request by gwyndol1ne**

### CODE REVIEW

File: src/test/kotlin/com/github/darderion/mundaneassignmentpolice/checker/SymbolRuleTests.kt [hunk 1]
```diff
@@ -64,6 +64,23 @@ class SymbolRuleTests: StringSpec({
 					if (area == SECTION) 3 else 0
 		}
 	}
+	"Symbol rule should detect incorrect using multiple links" {
+		SymbolRuleBuilder()
+			.symbol(']')
+			.ignoringAdjusting(*" ,".toCharArray())
+			.fromRight().shouldNotHaveNeighbor(*"[".toCharArray())
```
Comment (Darderion) [inline src/test/kotlin/com/github/darderion/mundaneassignmentpolice/checker/SymbolRuleTests.kt:71]: `*"[".toCharArray()` -> `'['`

File: src/test/kotlin/com/github/darderion/mundaneassignmentpolice/checker/SymbolRuleTests.kt [hunk 2]
```diff
@@ -94,5 +111,7 @@ class SymbolRuleTests: StringSpec({
 }) {
 	private companion object {
 		const val filePath = "${resourceFolder}checker/SymbolRuleTests.pdf"
+		const val filePath2 = "${resourceFolder}checker/MultipleLinksRuleTest.pdf"
+		const val filePath3 = "${resourceFolder}checker/LargeRussianLetterTests.pdf"
```
Comment (Darderion) [inline src/test/kotlin/com/github/darderion/mundaneassignmentpolice/checker/SymbolRuleTests.kt:115]: If a test file contains multiple PDF files then filePaths have to be just as descriptive as their file names. Furthermore, files have to match their tests by adhering to naming conventions already present in a project
(Example: https://github.com/Darderion/map/blob/master/src/test/kotlin/com/github/darderion/mundaneassignmentpolice/checker/ListRuleTests.kt)

File: src/test/resources/com/github/darderion/mundaneassignmentpolice/checker/.~lock.MultipleLinksRuleTest.pdf# [hunk 3]
```diff
@@ -0,0 +1 @@
+,gwyndol1ne,gwyndol1ne,18.02.2022 18:59,file:///home/gwyndol1ne/.config/libreoffice/4;
```
Comment (Darderion) [inline src/test/resources/com/github/darderion/mundaneassignmentpolice/checker/.~lock.MultipleLinksRuleTest.pdf#:1]: ?

File: src/test/kotlin/com/github/darderion/mundaneassignmentpolice/checker/SymbolRuleTests.kt [hunk 4]
```diff
@@ -64,6 +64,23 @@ class SymbolRuleTests: StringSpec({
 					if (area == SECTION) 3 else 0
 		}
 	}
+	"Symbol rule should detect incorrect using multiple links" {
```
Comment (Darderion) [inline src/test/kotlin/com/github/darderion/mundaneassignmentpolice/checker/SymbolRuleTests.kt:67]: "Symbol rule should detect incorrect using multiple links" -> "Symbol rule should detect incorrect use of multiple links"

File: src/test/kotlin/com/github/darderion/mundaneassignmentpolice/checker/SymbolRuleTests.kt [hunk 5]
```diff
@@ -64,6 +64,23 @@ class SymbolRuleTests: StringSpec({
...
+		SymbolRuleBuilder()
+			.symbol(']')
+			.ignoringAdjusting(*" ,".toCharArray())
+			.fromRight().shouldNotHaveNeighbor(*"[".toCharArray())
+			.getRule()
+			.process(PDFBox().getPDF(filePath2)).count() shouldBeExactly 3
+	}
+	"Symbol rule should detect incorrect using multiple links" {
```
Comment (Darderion) [inline src/test/kotlin/com/github/darderion/mundaneassignmentpolice/checker/SymbolRuleTests.kt:75]: Two tests with the same name

**Review by Darderion — CHANGES_REQUESTED** [review]
image
?

**Review by gwyndol1ne — COMMENTED** [review]
Updated rules

File: src/test/kotlin/com/github/darderion/mundaneassignmentpolice/checker/SymbolRuleTests.kt [hunk 6]
```diff
@@ -64,21 +64,21 @@ class SymbolRuleTests: StringSpec({
...
-			.ignoringAdjusting(*" ,".toCharArray())
-			.fromRight().shouldNotHaveNeighbor(*"[".toCharArray())
+			.ignoringAdjusting(',',' ')
+			.fromRight().shouldNotHaveNeighbor('[')
 			.getRule()
-			.process(PDFBox().getPDF(filePath2)).count() shouldBeExactly 3
+			.process(PDFBox().getPDF(filePathMultipleLinks)).count() shouldBeExactly 3
 	}
-	"Symbol rule should detect incorrect using multiple links" {
+	"Symbol rule should detect incorrect using capital letters" {
```
Comment (Darderion) [inline src/test/kotlin/com/github/darderion/mundaneassignmentpolice/checker/SymbolRuleTests.kt:75]: "usage of letters"

**Pull request merged**